Material.Icons.Avalonia/MaterialIcon.axaml.cs

```diff
@@ -35,5 +40,95 @@ public MaterialIconAnimation Animation {
             get => GetValue(AnimationProperty);
             set => SetValue(AnimationProperty, value);
         }
+
+        public static readonly DirectProperty<MaterialIcon, GeometryDrawing> DrawingProperty =
+            AvaloniaProperty.RegisterDirect<MaterialIcon, GeometryDrawing>(
+                nameof(Drawing),
+                o => o.Drawing);
+
+        /// <summary>
+        /// Gets the <see cref="GeometryDrawing"/> of the icon.
+        /// </summary>
+        public GeometryDrawing Drawing { get; } = new();
+
+        // Default size for Material Icons
+        private static readonly Rect DefaultIconBounds = new(0, 0, 24, 24);
+
+        #endregion
+
+        #region Constructor
+
+        public MaterialIcon() {
+            Drawing.Brush = Foreground;
+        }
+
+        #endregion
+
+        #region Overrides
+
+        /// <inheritdoc />
+        protected override void OnLoaded(RoutedEventArgs e) {
+            if (Drawing.Geometry is null)
+                SetGeometry();
+            base.OnLoaded(e);
+        }
+
+        /// <inheritdoc />
+        protected override void OnPropertyChanged(AvaloniaPropertyChangedEventArgs e) {
+            base.OnPropertyChanged(e);
+
+            if (ReferenceEquals(e.Property, KindProperty)) {
```

Comment:
We dont need ReferenceEquals here, just `e.Property == KindProperty` are fine

Reply:
Can change to ==.

What about `IconBrush` vs `IconColor` vs `IconForeground` ?